streams/src/test/java/org/apache/kafka/streams/state/internals/NamedCacheTest.java

```diff
@@ -83,18 +83,15 @@ public void testMetrics() {
         metricTags.put("task-id", taskIDString);
         metricTags.put("client-id", "test");
 
-        assertNotNull(streamMetrics.registry().getSensor("hitRatio"));
```

Comment:
See my earlier comment on not testing for sensor names for the rationale.

Reply:
So my only question is if we can't or didn't create the metric we would not be able to create a sesnsor? I'm guessing the answer is yes, so the changes for sensor testing makes sense to me.

Reply:
Right, the workflow is:
1. create a sensor
2. add metrics to the sensor

So, if we didn't create the sensor, we couldn't add the metrics.

And if we don't add the metrics for this reason or any other, the tests will catch it, because they are verifying the metrics exist.